src/mkdocstrings_handlers/python/templates/material/_base/docstring.html

```diff
@@ -1,27 +1,27 @@
 {% if docstring_sections %}
   {{ log.debug("Rendering docstring") }}
   {% for section in docstring_sections %}
-    {% if section.kind.value == "text" %}
+    {% if config.show_docstring_description and section.kind.value == "text" %}
       {{ section.value|convert_markdown(heading_level, html_id) }}
-    {% elif section.kind.value == "attributes" %}
+    {% elif config.show_docstring_section_attributes and section.kind.value == "attributes" %}
       {% include "docstring/attributes.html" with context %}
-    {% elif section.kind.value == "parameters" %}
+    {% elif config.show_docstring_section_parameters and section.kind.value == "parameters" %}
       {% include "docstring/parameters.html" with context %}
-    {% elif section.kind.value == "other parameters" %}
+    {% elif config.show_docstring_section_other_parameters and section.kind.value == "other parameters" %}
       {% include "docstring/other_parameters.html" with context %}
-    {% elif section.kind.value == "raises" %}
+    {% elif config.show_docstring_section_raises and section.kind.value == "raises" %}
       {% include "docstring/raises.html" with context %}
-      {% elif section.kind.value == "warns" %}
+    {% elif config.show_docstring_section_warns and section.kind.value == "warns" %}
         {% include "docstring/warns.html" with context %}
```

Comment:
Seems like we can unindent here as well, for consistency.

Reply:
yep sharp eye, thanks!